pkg/events/events.go

```diff
@@ -0,0 +1,34 @@
+package events
+
+import (
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/tools/record"
+)
+
+// Event message format "medik8s <operator shortname> <message>"
+const format = "medik8s %3s %s"
+
+type EventRecorder struct {
```

Comment:
IIUC this struct implements the interface of `record.EventRecorder`
it also adds 4 additional methods (NormalEvent,  WarningEvent , etc... )

- I think it should also override the original methods of `record.EventRecorder` with our formatted message.
- Nit: I'd consider returning an interface instead of a struct